contrib/pyln-testing/pyln/testing/utils.py

```diff
@@ -1280,12 +1284,20 @@ def get_nodes(self, num_nodes, opts=None):
 
         assert len(opts) == num_nodes
 
+        # Only trace one random node's plugins, to avoid OOM.
+        if SLOW_MACHINE:
+            valgrind_plugins = [False] * num_nodes
+            valgrind_plugins[random.randint(0, num_nodes-1)] = True
```

Comment:
```suggestion
            valgrind_plugins[random.randint(0, num_nodes - 1)] = True
```

I suspect that this is the cause of the CI failure (sadly)